app/Rules/ValidTestQuestionsScore.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+
+class ValidTestQuestionsScore implements Rule
+{
+    /**
+     * Create a new rule instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function passes($attribute, $value)
+    {
+        $score = 0;
+
+        foreach ($value as $question) {
+            $score += $question['score'];
```

Comment:
This will throw an error if you'll not provide the score with the questions in the request.